src/renderer/components/AbsolutePositioningHelper/styles.module.scss

```diff
@@ -1,4 +1,6 @@
+@import '../../../../scss/variables';
+
 .absolutePositioningHelper {
   position: fixed;
-  z-index: 1000;
+  z-index: map-get($z-index, absolutePositioningHelper);
```

Comment:
I'd use the same convention for all keys, e.g. `'absolute-positioning-helper'`. I realize this class isn't named the same way, but we can change this later.